src/source/SwiftBridgingHeaderGenerator.scala

```diff
@@ -38,8 +38,23 @@ class SwiftBridgingHeaderGenerator(spec: Spec) extends Generator(spec) {
 }
 
 object SwiftBridgingHeaderGenerator {
-  def writeAutogenerationWarning(writer: Writer) {
+  
+  val bridgingVarPrefix = (s: String) => s.split('-').mkString("_")
```

Comment:
Similar to my comment below, I'd prefer to remove the `Prefix` here, to make it something like `bridgingHeaderName`, since it doesn't have to be used as a prefix. (Although it's not a great name either, as `-` is replaced by `_` and this method name I suggest wouldn't make it obvious.)